configuration/micro_kernel_trait.rst

```diff
@@ -102,6 +102,44 @@ that define your bundles, your services and your routes:
     ``RouteCollectionBuilder`` has methods that make adding routes in PHP more
     fun. You can also load external routing files (shown below).
 
+Adding Interfaces to "Micro" Kernel
+-----------------------------------
+
+When using the ``MicroKernelTrait``, you can also implement the
```

Comment:
```suggestion
When using the :class:`.......MicroKernelTrait`, you can also implement the
```
?

to make it even more discoverable?

Reply:
Not sure it's needed at this point, the link is made at the beginning of the article and line 88 it is already repeated without.
So unless we update both I would leave it at this.